Resources/ServerInfo/Guidebook/Mobs/Vox.xml

```diff
@@ -5,12 +5,22 @@
     <GuideEntityEmbed Entity="MobVox" Caption=""/>
   </Box>
 
-  [color=#ffa500]Caution! This species has a severely limiting game mechanic and is not recommended for new players. [/color]
+  [color=#ffa500]Warning! This species is not recommended for new players due to their fatal allergy to oxygen![/color]
 
-  They breathe nitrogen, and [color=#ffa500] oxygen is very toxic to them.[/color]
-  They need to be on internals at all times to avoid being poisoned by the station's oxygen.
-  Eating or drinking will require you to remove your breath mask, although injecting liquid food is an option.
+  Vox breathe nitrogen, and [color=#ffa500] oxygen is toxic to them.[/color]
+  Unfortunately, space stations tend to be full of oxygen,
+  so Vox must use an internal nitrogen supply at almost all times to avoid fatal exposure.
```

Comment:
```suggestion
  Unfortunately, space stations tend to be full of oxygen, so Vox must use an internal nitrogen supply at almost all times to avoid fatal exposure.
```

Reply:
I think it's fine as is. If you feel strongly about it, submit a PR with the change to my working branch, and I promise I will genuinely consider it, so long as there is time to do so before merge. But that could happen at any moment